mypy/treetransform.py

```diff
@@ -392,7 +392,7 @@ def visit_reveal_type_expr(self, node: RevealTypeExpr) -> RevealTypeExpr:
         return RevealTypeExpr(self.expr(node.expr))
 
     def visit_super_expr(self, node: SuperExpr) -> SuperExpr:
-        new = SuperExpr(node.name)
+        new = SuperExpr(node.name, cast(CallExpr, self.expr(node.call)))
```

Comment:
Why a cast needed here? Is it possible to replace it with an assert?

Reply:
Replaced with an assert.